spring-cloud-stream/src/main/java/org/springframework/cloud/stream/config/ChannelBindingServiceProperties.java

```diff
@@ -122,7 +122,7 @@ public Properties getConsumerProperties(String inputChannelName) {
 				channelConsumerProperties.setProperty(BinderPropertyKeys.CONCURRENCY,
 						Integer.toString(bindingProperties.getConcurrency()));
 			}
-			if (bindingProperties.isDurableSubscription() != null) {
+			if (Boolean.TRUE.equals(bindingProperties.isDurableSubscription())) {
```

Comment:
Well, the discussion on durability is a longer one, but as a norm, I think it adds some confusion if we filter out properties based on their value, before letting the binder to act on it.